src/openrct2/ride/TrackData.cpp

```diff
@@ -5466,6 +5863,16 @@ static int32_t EvaluatorQuarterTurn3Tiles(const int16_t progress)
     return -(progress / 2) + 134;
 }
 
+static int32_t EvaluatorMediumHalfLoopUp(const int16_t progress)
+{
+    return ((static_cast<uint16_t>(-(progress - 244))) / 4) + 51;
```

Comment:
This cast looks mighty suspect what are you trying to achieve here

Reply:
I put the cast there because the large/small half loops have it, but I couldn't figure out what it was actually there for, so I think it would be fine to just remove it.